### PR TITLE
API 4321 PGD Production Monitoring

### PR DESCRIPTION
**Pull request by that-guy-is-it**

### CODE REVIEW

File: pingdom-checks/patient-generated-data.ping [hunk 1]
```diff
@@ -10,3 +10,12 @@ pingdom save-check \
...
   -a integrationids_csv="$PGD_SLACK_ID"
+
+pingdom save-check \
+  --template fhir-resource \
+  -a name=production-pgd-r4-questionnaire \
+  -a host=api.va.gov \
+  -a url="/services/pgd/v0/r4/Questionnaire/37953b72-961b-41ee-bd05-86c62bacc46b" \
+  -a authorization_token="$HEALTH_APIS_STATIC_ACCESS_TOKEN" \
+  -a userids_csv="$STATUSPAGE_PRODUCTION_FHIR" \
+  -a integrationids_csv="$PGD_SLACK_ID"
```
Comment (jblefkove-va) [inline pingdom-checks/patient-generated-data.ping:21]: newline eof

File: pingdom-checks/patient-generated-data.ping [hunk 2]
```diff
@@ -10,3 +10,12 @@ pingdom save-check \
...
   -a userids_csv="$NO_USERS" \
   -a integrationids_csv="$PGD_SLACK_ID"
+
+pingdom save-check \
+  --template fhir-resource \
+  -a name=production-pgd-r4-questionnaire \
+  -a host=api.va.gov \
+  -a url="/services/pgd/v0/r4/Questionnaire/37953b72-961b-41ee-bd05-86c62bacc46b" \
+  -a authorization_token="$HEALTH_APIS_STATIC_ACCESS_TOKEN" \
+  -a userids_csv="$STATUSPAGE_PRODUCTION_FHIR" \
```
Comment (jblefkove-va) [inline pingdom-checks/patient-generated-data.ping:20]: hmm I don't think we want this on statuspage yet

File: pingdom-checks/patient-generated-data.ping [hunk 3]
```diff
@@ -10,3 +10,12 @@ pingdom save-check \
...
   -a userids_csv="$NO_USERS" \
   -a integrationids_csv="$PGD_SLACK_ID"
+
+pingdom save-check \
+  --template fhir-resource \
+  -a name=production-pgd-r4-questionnaire \
+  -a host=api.va.gov \
+  -a url="/services/pgd/v0/r4/Questionnaire/37953b72-961b-41ee-bd05-86c62bacc46b" \
+  -a authorization_token="$HEALTH_APIS_STATIC_ACCESS_TOKEN" \
+  -a userids_csv="$NO_USERS" \
```
Comment (bschofield-va) [inline pingdom-checks/patient-generated-data.ping:20]: ^ will this eventually be hooked up to Statuspage?

Comment (jblefkove-va) [reply pingdom-checks/patient-generated-data.ping:20]: Most likely, I'll check with Gabe

**Pull request merged**